gui/executionGraphGui/client/apps/eg/src/app/components/file-browser/file-browser.component.ts

```diff
@@ -110,16 +146,38 @@ export class FileBrowserComponent implements OnInit {
     });
   }
 
+  private saveFile(path: string, checkOverwrite: boolean) {
+    assertThat(this.mode === FileBrowserMode.Save);
+
+    const showOverwrite = checkOverwrite ? this.checkOverwrite(path) : false;
+
+    let save = false;
+
+    const dialogRef = this.dialog.open(ConfirmationDialogComponent, {
+      minWidth: '10%',
+      data: {
+        title: showOverwrite ? 'Overwrite?' : 'Save?',
+        question: showOverwrite ? `Do you really want to overwrite the path '${path}' ?` : `Saving to path: '${path}' ?`
+      }
+    });
+    dialogRef.afterClosed().subscribe(result => {
+      if (result) {
+        save = true;
+      }
+    });
+
+    if (save) {
```

Comment:
Does that work? Shouldn't this belong in the subscription above?

Reply:
Ui, but jeah plain wrong ;-) :+1: